tests/python/relay/test_ir_text_printer.py

```diff
@@ -284,5 +284,12 @@ def test_optional_info():
     assert txt.count("/* ty=int32 */") == 3
 
 
+def test_slash_in_identifier():
+    x = relay.var("base/x")
+    y = relay.var("base/y")
+    z = x + y
+    txt = astext(z)
```

Comment:
You might need to check the output of `astext` to make sure the names are preserved.

Reply:
Yes, inside `astext()` it is tested. 
[test_ir_text_printer.py#L39](https://github.com/apache/tvm/blob/f82cf36c12d964052bf11830b5180bcee051266c/tests/python/relay/test_ir_text_printer.py#L39)
I just reuse it here.

Reply:
Done